### PR TITLE
Feature/wav 1463

### PR DESCRIPTION
**Pull request by junggren**

### CODE REVIEW

File: docs/api/data-usage-api.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Get the data usage of a company in a month, as a total value in bytes.
+
+**Endpoint:** `GET /usage/data/{year}/{month}`
+
+**Request Query Parameters**
+
+| Field        | Type    | Description                | Required |
+| ------------ | ------- | -------------------------- | -------- |
+| year         | Integer | Year to see data usage in | Yes       |
+| month        | String  | Month, in text, to see consumption for                  | Yes       |
```
Comment (clissau) [inline docs/api/data-usage-api.md]: Could be nice to align the vertical dividers, and limit the white space of descriptions.

File: docs/api/data-usage-api.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Get the data usage of a company in a month, grouped by SIM card groups.
+
+**Endpoint:** `GET /usage/data/collections/{year}/{month}`
+
+**Response Payload**
+
+| Field        | Type    | Description                | Required |
+| ------------ | ------- | -------------------------- | -------- |
+| year         | Integer | Year to see data usage in | Yes       |
+| month        | String  | Month, in text, to see consumption for                  | Yes       |
```
Comment (clissau) [inline docs/api/data-usage-api.md]: Same as above

File: docs/api/invoice-api.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+If the month is current not-finished month, the invoice will be the most recent snapshot.
+
+**Endpoint:** `GET /invoice/{year}/{month}`
+
+**Request Query Parameters**
+
+| Field        | Type    | Description                | Required |
+| ------------ | ------- | -------------------------- | -------- |
+| year         | Integer | Year for the invoice in ISO 8601 format      | Yes       |
+| month        | String  | Month for invoice, fx "may"| Yes       |
```
Comment (clissau) [inline docs/api/invoice-api.md]: Same as above

Comment (clissau) [reply docs/api/invoice-api.md]: Suggestion to align the descriptions for year and month, compared to data-usage-api.md

File: docs/general-information/data-types.md [hunk 4]
```diff
@@ -178,3 +178,86 @@ INGOING_SMS     | Ingoing sms service enabled
...
+
+Simple representation of an invoice
+
+| Field             | Type                                                                      |
+| ---------------   | ---------------                                                           |
+| invoiceNumber               | Long                                                                    |
+| startDate            | ISO 8601 Date                                                                    |
+| endDate             | ISO 8601 Date                  |
+| currencyCode  | String    |
+| invoiceLines      | List([InvoiceLine](/general-information/data-types#BasicInvoiceLineDTO))
```
Comment (clissau) [inline docs/general-information/data-types.md:192]: Same as above

File: docs/general-information/data-types.md [hunk 5]
```diff
@@ -178,3 +178,86 @@ INGOING_SMS     | Ingoing sms service enabled
...
+| quantity               | Number                                                                    |
+| fromType            | [BillableType](/general-information/data-types#BillableType) |
+| toType            | [BillableType](/general-information/data-types#BillableType) |
+| fromCountry             | [CountryCode](/general-information/data-types#CountryCode) |
+| toCountry             | [CountryCode](/general-information/data-types#CountryCode) |
+| direction  | [Directions](/general-information/data-types#Directions) |
+| productName      | String |
+| chargeableName      | String |
+| chargeablePackageName      | String |
+| description      | String |
```
Comment (clissau) [inline docs/general-information/data-types.md:210]: Same as above

**Review by grandegiraffo — CHANGES_REQUESTED** [review]
Add the new pages to `mkdocs.yml`

File: docs/api/data-usage-api.md [hunk 6]
```diff
@@ -49,10 +49,10 @@ List([DataUsageBySimCardCollections](/general-information/data-types/#DataUsageB
...
-| collectionId | UUID    | Unique ID for the SIM card collection we want data usage for | Yes       |
-| year         | Integer | Year to see data usage in | Yes       |
-| month        | String  | Month, in text, to see consumption for                  | Yes       |
+| Field        | Type    | Description                                                  | Required |
+| ------------ | ------- | ------------------------------------------------------------ | ---------|
+| collectionId | UUID    | Unique ID for the SIM card collection we want data usage for | Yes      |
+| year         | Integer | Year to see data usage in                                    | Yes      |
+| month        | String  | Month, in text, to see consumption for                       | Yes      |
 
 [DataUsageBySimCardCollections](/general-information/data-types/#DataUsageBySimCardCollections)
```
Comment (clissau) [inline docs/api/data-usage-api.md]: `#DataUsageBySimCardCollections` must be in lower case. Else the link to section will not work properly.

File: docs/api/data-usage-api.md [hunk 7]
```diff
@@ -34,10 +34,10 @@ Get the data usage of a company in a month, grouped by SIM card groups.
...
-| Field        | Type    | Description                | Required |
-| ------------ | ------- | -------------------------- | -------- |
-| year         | Integer | Year to see data usage in | Yes       |
-| month        | String  | Month, in text, to see consumption for                  | Yes       |
+| Field        | Type    | Description                              | Required |
+| ------------ | ------- | -----------------------------------------| -------- |
+| year         | Integer | Year to see data usage in                | Yes      |
+| month        | String  | Month, in text, to see consumption for   | Yes      |
 
 List([DataUsageBySimCardCollections](/general-information/data-types/#DataUsageBySimCardCollections))
```
Comment (clissau) [inline docs/api/data-usage-api.md]: #DataUsageBySimCardCollections must be in lower case. Else the link to section will not work properly.

**Review by clissau — CHANGES_REQUESTED** [review]

**Review by clissau — APPROVED** [review]

**Pull request merged**